waiter/integration/waiter/token_request_test.clj

```diff
@@ -662,6 +662,22 @@
       (is (not (str/includes? body "clojure")) body)
       (is (str/includes? body "The following environment variable keys are reserved: HOME.") body))))
 
+(deftest ^:parallel ^:integration-fast test-token-parameters-exceed-limits
+  (testing-using-waiter-url
+    (let [constraints (setting waiter-url [:service-description-constraints])
+          max-constraints (->> constraints
```

Comment:
`extract-max-constraints`?

Reply:
Oops, fixed. Do you want to wait for the build?